rest/route/admin_routes.go

```diff
@@ -339,3 +339,53 @@ func (h *restartHandler) Execute(ctx context.Context, sc data.Connector) (Respon
 		Result: []model.Model{restartModel},
 	}, nil
 }
+
+func getRevertRouteManager(route string, version int) *RouteManager {
+	rh := revertHandler{}
+	handler := MethodHandler{
+		PrefetchFunctions: []PrefetchFunc{PrefetchUser},
+		Authenticator:     &SuperUserAuthenticator{},
+		RequestHandler:    rh.Handler(),
+		MethodType:        http.MethodPost,
+	}
+
+	return &RouteManager{
+		Route:   route,
+		Methods: []MethodHandler{handler},
+		Version: version,
+	}
+}
+
+type revertHandler struct {
+	GUID string `json:"guid"`
+}
+
+func (h *revertHandler) Handler() RequestHandler {
+	return &revertHandler{}
+}
+
+func (h *revertHandler) ParseAndValidate(ctx context.Context, r *http.Request) error {
+	if err := util.ReadJSONInto(r.Body, h); err != nil {
+		return err
+	}
+	defer r.Body.Close()
```

Comment:
I don't usually close bodies in ParseAndValidate. I think this is handled by a higher level.

Reply:
I took a look at the implementation in handler.go and didn't see that this is closed